pkgs/applications/networking/libcoap/default.nix

```diff
@@ -1,26 +1,29 @@
-{ fetchFromGitHub, automake, autoconf, which, pkg-config, libtool, lib, stdenv }:
+{ fetchFromGitHub, automake, autoconf, which, pkg-config, libtool, lib, stdenv, gnutls, asciidoc, doxygen
+, tls ? true
+, docs ? true
```

Comment:
I think we normally call these something like `withTLS` or `enableTLS`, although I'm not sure there is really any agreed-upon reason for that.

Reply:
Yes, I think "with" is slightly more common.

Reply:
I'm using `with` now.